phase2/phase2/phase2/Processor/InvertedIndexManager/InvertedIndexBuilder.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using phase2.Models;
+
+namespace phase2.InvertedIndexManager;
+
+public class InvertedIndexBuilder
+{
+    public static Dictionary<string, List<string>> BuildInvertedIndex(List<DataFile> docs)
+    {
+        var invertedData = new Dictionary<string, List<string>>();
+
+        foreach (var element in docs)
+        {
+            var words = element.Data.Split(' ', StringSplitOptions.RemoveEmptyEntries);
+            foreach (var elementWord in words)
+                if (!invertedData.ContainsKey(elementWord))
+                {
+                    var values = new List<string>();
+                    values.Add(element.FileName);
```

Comment:
initialize on declaration

Reply:
https://giannisakritidis.com/blog/Lists-in-csharp/#:~:text=UPDATE%3A%20From%20.,items%2C%20but%20so%20do%20lists.

var values = new List<string>()
{
     element.fileName
}